view.rb

```diff
@@ -19,4 +19,13 @@ def ask_for_name
     puts "What is the animal's name?"
     return gets.chomp
   end
-end
\ No newline at end of file
+
+  def ask_for_animal_index
+    puts 'Which animal would you like to adopt? Key in the index number.'
+    return gets.chomp.to_i
+  end
+
+  def adopted_animal(animal)
```

Comment:
I think this method name could be a bit more descriptive. Maybe `show_post_adoption_message`? Or `adoption_thank_you_message`?